src/implementations/react/src/elements/components/FormElements/Dropdown.js

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from 'react';
+import DropdownAdapter
+  from '../../../adapters/FormElements/DropdownAdapter.js';
+import * as PropTypes from 'prop-types';
+
+const OptionAdapter = DropdownAdapter.Option;
+
+class Option extends Component {
```

Comment:
I made Option a class because calling `bind` within a render function has performance issues.

https://daveceddia.com/react-best-alternative-bind-render/
*(I haven't read this, just came up in a quick google on the topic)*